test/usage.js

```diff
@@ -58,6 +81,28 @@ describe('usage tests', function () {
           r.logs.should.have.length(0)
           r.exit.should.be.ok
         })
+        it('fails if one command and an argument not on the list are provided', function () {
```

Comment:
could you add two additional test cases:

* one that verifies that it fails if all the required arguments exist, but not enough commands are provided?
* one that verifies that no failure occurs if the required arguments and the required number of commands are provided.